Aquarius/Aquarius.cpp

```diff
@@ -62,22 +68,23 @@ namespace Aquarius {
         glBindVertexArray(VA0);
 
         // Create and bind vertex buffer
-        VertexBuffer VB0 = VertexBuffer(data.data(), data.size() * sizeof(data[0]));
+        VertexBuffer VB0 = VertexBuffer(data.data(), data.size() * sizeof(float));
```

Comment:
This is very minor, but if you're gonna invoke the constructor on the RHS of the expression like that, i'm partial to using `auto VB0` instead, cause the type is still very evident based off of the variable name and the constructor

Reply:
I will definitely do that, sick point